README.md

```diff
@@ -171,6 +174,8 @@ For other architectures, refer to [build](https://github.com/DMTF/libspdm/blob/m
 
 2) [OpenSSL](https://www.openssl.org) as cryptography library. Version 3.0.14.
 
+3) [OQS](https://github.com/open-quantum-safe/liboqs) as PQC cryptography library. Version 0.12.0.

```

Comment:
Is this intended to be a long-lived cryptography library, or just one that will be used for development and eventually MbedTLS and OpenSSL will support PQC in production?

Reply:
I would say: If MbedTLS and OpenSSL support PQC in production, we can remove OQS for sure.

But before that, we have to use OQS for development. And I don't know **WHEN** the mbedTLS or OpenSSL will be ready.

Reply:
I added clarification.